traitsui/editors/video_editor.py

```diff
@@ -40,45 +40,61 @@ class VideoEditor(BasicEditorFactory):
     aspect_ratio = AspectRatio()
 
     #: The name of a trait to synchronise with the audio muted state of
-    #: the video.  The referenced trait should be a Bool.
-    muted = Str(sync_value='from', sync_name='muted')
+    #: the video.
+    muted_name = Str()
+    #: True if the audio is muted, False otherwise
+    muted = Bool(sync_value='from', sync_name='muted_name')
 
     #: The name of a trait to synchronise with the audio volume of
-    #: the video player.  The referenced trait should be a Range(0.0, 100.0).
-    #: Values are on a logarithmic scale.
-    volume = Str(sync_value='from', sync_name='volume')
+    #: the video player.
+    volume_name = Str()
+    #: Audio volume on a logarithmic scale
+    volume = Range(0.0, 100.0, 50.0,
+                   sync_value='from', sync_name='volume_name')
 
     #: The name of a trait to synchronise with the playback rate of
-    #: the video.  The referenced trait should be a Float.
-    playback_rate = Str(sync_value='from', sync_name='playback_rate')
+    #: the video.
+    playback_rate_name = Str()
+    #: The playback speed of the video. Negative values are allowed but may not
+    #: be supported by the underlying implementation.
+    playback_rate = Float(sync_value='from', sync_name='playback_rate_name')
```

Comment:
I think that this is the right idea for these.  It can make sense to set an initial default which is never modified and so doesn't need a bound trait.